#! /bin/sh -e
## 10_path_changes.dpatch by Jose Carlos Garcia Sogo <jsogo@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Used to change paths from upstream ones to paths used in a Debian system.

if [ $# -lt 1 ]; then
    echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch} ${2:+-d $2}"

case "$1" in
    -patch) patch -p1 ${patch_opts} < $0;;
    -unpatch) patch -R -p1 ${patch_opts} < $0;;
    *)
        echo "`basename $0`: script expects -patch|-unpatch as argument" >&2
        exit 1;;
esac

exit 0

@DPATCH@
diff -urNad smokeping-1.37/bin/smokeping.dist /tmp/dpep.QzUYax/smokeping-1.37/bin/smokeping.dist
--- smokeping-1.37/bin/smokeping.dist	2005-01-29 22:22:46.000000000 +0100
+++ /tmp/dpep.QzUYax/smokeping-1.37/bin/smokeping.dist	2005-01-29 22:30:12.000000000 +0100
@@ -1,12 +1,12 @@
-#!/usr/sepp/bin/perl-5.8.0 -w
+#!/usr/bin/perl -w
 # -*-perl-*-
 
-use lib qw(/usr/pack/rrdtool-1.0.47-to/lib);
-use lib qw(/home/oetiker/data/projects/AADJ-smokeping/dist/lib);
+use lib qw(/usr/share/perl5/smokeping);
+use lib qw(/usr/share/perl5);
 
 use Smokeping 1.38;
 
-Smokeping::main("etc/config.dist");
+Smokeping::main("/etc/smokeping/config");
 
 =head1 NAME
 
diff -urNad smokeping-1.37/htdocs/smokeping.cgi.dist /tmp/dpep.QzUYax/smokeping-1.37/htdocs/smokeping.cgi.dist
--- smokeping-1.37/htdocs/smokeping.cgi.dist	2005-01-29 22:22:46.000000000 +0100
+++ /tmp/dpep.QzUYax/smokeping-1.37/htdocs/smokeping.cgi.dist	2005-01-29 22:30:12.000000000 +0100
@@ -1,12 +1,11 @@
-#!/usr/sepp/bin/speedy -w
+#!/usr/bin/speedy -w
 # -*-perl-*-
 
-use lib qw(/usr/pack/rrdtool-1.0.33-to/lib/perl);
-use lib qw(/home/oetiker/data/projects/AADJ-smokeping/dist/lib);
+use lib qw (/usr/share/perl5/smokeping);
 
 use Smokeping 1.38;
 
-Smokeping::cgi("/home/oetiker/data/projects/AADJ-smokeping/dist/etc/config");
+Smokeping::cgi("/etc/smokeping/config");
 
 
 BEGIN {
diff -urNad smokeping-1.37/lib/probes/FPing6.pm /tmp/dpep.QzUYax/smokeping-1.37/lib/probes/FPing6.pm
--- smokeping-1.37/lib/probes/FPing6.pm	2005-01-29 22:22:46.000000000 +0100
+++ /tmp/dpep.QzUYax/smokeping-1.37/lib/probes/FPing6.pm	2005-01-29 22:30:12.000000000 +0100
@@ -8,7 +8,7 @@
 
  *** Probes ***
  + FPing6
- binary = /usr/sbin/fping6
+ binary = /usr/bin/fping6
 
 =head1 DESCRIPTION
 
diff -urNad smokeping-1.37/lib/probes/FPing.pm /tmp/dpep.QzUYax/smokeping-1.37/lib/probes/FPing.pm
--- smokeping-1.37/lib/probes/FPing.pm	2005-01-29 22:22:46.000000000 +0100
+++ /tmp/dpep.QzUYax/smokeping-1.37/lib/probes/FPing.pm	2005-01-29 22:30:12.000000000 +0100
@@ -8,7 +8,7 @@
 
  *** Probes ***
  + FPing
- binary = /usr/sepp/bin/fping
+ binary = /usr/bin/fping
  packetsize = 1024
 
 =head1 DESCRIPTION
